meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-development.bb

```diff
@@ -25,6 +25,7 @@ PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " python3-pygobject"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " gobject-introspection"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " e2fsprogs"
 PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " mbed-crypto-test"
+PACKAGEGROUP_MBL_DEVELOPMENT_PKGS_append = " psa-storage"
```

Comment:
This needs to be optional in the build so we can turn it off for 0.8 as the repo is not yet approved and public.

Reply:
Fixed.